libs/src/api/serviceProvider.js

```diff
@@ -79,7 +81,7 @@ class ServiceProvider extends Base {
       blacklist
     })
 
-    const { primary, secondaries, services } = await creatorNodeSelection.select()
+    const { primary, secondaries, services } = await creatorNodeSelection.select(performSyncCheck)
```

Comment:
sync checks are turned off for assigning a replica set to users on signup: https://github.com/AudiusProject/audius-protocol/pull/1142/files#diff-a34533690240ad151f75e407a02a875ee287c63c60bbdf814955e6ecbce52b13R237-R241

Reply:
in response to sid @SidSethi's comments:
1. yup! the sync check prior to this change always occurred. now that we are adding `autoSelectCreatorNodes` to new users on signup, we do not need to do any sync checks because there will be no syncing occurring. 
2. so actually in this task, I updated the [client code](https://github.com/AudiusProject/audius-client/pull/159/files#diff-0e9d4bad4ac3bb8689f42bf0d0cdb10fd8225ee4a02f12fe133b9430243289ebR1041-R1053) so that users who get assigned a replica set immediately during signup and then subsequently upload tracks only have their `is_creator` flag set to true. existing users will still go through the `upgradeToCreator` flow and have their data synced from UM -> assigned replica set

Reply:
UPDATE TO THIS: 
- actually removed the `updateIsCreatorToTrue`, and updated `upgradeToCreator` to handle setting the `is_creator` to `true`
- this is to decrease the surface area of interface changes, especially on the dapp

Reply:
this all sounds good, i believe we can resolve this thread